server/lib/configuration.js

```diff
@@ -149,6 +149,32 @@ var conf = module.exports = convict({
     doc: 'Google Analytics id',
     format: String
   },
+  hpkp_config: {
+    enabled: {
+      default: true,
```

Comment:
We have moved to setting default development values in lib/configuration.js. Will setting the default to `true` with the actual SHAs will break local and fxa-dev development?